docs/content/advanced/backend/plugins/create.md

```diff
@@ -266,6 +266,29 @@ Finally, start your server and test your plugin’s functionalities:
 npm run start
 ```
 
+### Testing Troubleshooting
+
+ If you are certain your plugin is structured correctly and you encounter `error: The class must be a valid service implementation` then you can try the following:
+
+```bash npm2yarn
+cd /your/medusa-backend/node_modules/medusa-interfaces
+npm link
+cd /your/plugin/directory
+rm -rf node_modules/medusa-interfaces
+npm link medusa-interfaces
+npm link
+cd /your/medusa-backend
+npm link your-plugin
+```
+
+ This will link the `medusa-interfaces` package from your `medusa-backend` to your plugin directory and then link your plugin to your `medusa-backend` and should fix the error.
```

Comment:
Add before this a message to explain that `<SERVER_PATH>` and `<PLUGIN_PATH>` should be replaced with their actual values. It can be something like "Where `<SERVER_PATH>` is the path to your Medusa server and `<PLUGIN_PATH>` is the path to your plugin."